verification/src/http_server/handlers/verification/sourcify/metadata.rs

```diff
@@ -0,0 +1,180 @@
+use std::collections::BTreeMap;
+
+use ethers_solc::EvmVersion;
+use serde::Deserialize;
+
+use crate::VerificationResult;
+
+use super::types::Files;
+
+const METADATA_FILE_NAME: &str = "metadata.json";
+
+// There is struct for metadata in ethers_solc::artifacts::Metadata
+// however it
+//
+#[derive(Debug, PartialEq, Deserialize)]
+pub struct Metadata {
+    pub settings: MetadataSettings,
+    pub compiler: Compiler,
+    pub output: Output,
+}
+
+#[derive(Debug, PartialEq, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct MetadataSettings {
+    pub compilation_target: BTreeMap<String, String>,
+    pub optimizer: Optimizer,
+    pub libraries: BTreeMap<String, String>,
+    #[serde(rename = "camelCase")]
```

Comment:
Is there any reason to put macros once again while you have `rename_all` above the structure?

Reply:
Get you. Do you have any idea why it is the case?

Reply:
rename_all renames all fields in struct `MetadataSettings` (evm_version -> evmVersion)
but it doesn't (and shouldn't) apply this rule to every type of fields (e.g. for `EvmVersion`)

I think this reasonable, maybe I don't want this kind of behavior